### PR TITLE
Created binary_search.java

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

**Review by khanchi97 — COMMENTED** [review]
I'm in suspect that you have just copy/pasted this code.If what I'm saying is right then please write your code on your own otherwise it will be complained to KWoC.
Also don't forget to add a README file also in your implentations.

**Review by khanchi97 — CHANGES_REQUESTED** [review]
Make required changes.

**Comment by sidntrivedi** [comment]
Sir,

With all due respect, I have not copied it.I have taken variables like c,
middle etc. so that the readers can get a better understanding of the
concept. I had seen the other method of binary search in a website, so I
mentioned that too for extra information. But still, I am making some
changes. Thank you. Sorry for the inconvenience.

**Comment by khanchi97** [comment]
okay, no issue.Just wait for collaborators to review this.
@RajaVamsi11 ^

**Comment by khanchi97** [comment]
And make a different README file to describe what it does instead adding it to the code, and try to explain how this code is working :)

**Comment by sidntrivedi** [comment]
Sir,

Added Readme.txt for Binary Search Algo. Kindly review. Regards.

File: search-algos/binary_search/binary_search.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.*;
+class binary_search 
+{
+  public static void main(String args[])
+  {
+    int n,c,mid,beg,end,value,i,position;
+    Scanner in = new Scanner(System.in);
+    System.out.println("Enter number of elements");
+    n = in.nextInt();                       //inputting the length of array
+    int numbers = new int[n];               //declaring array of n numbers
```
Comment (khanchi97) [inline search-algos/binary_search/binary_search.java]: See the syntax error here.
change this line to
`int [] numbers = new int[n];`

**Review by khanchi97 — CHANGES_REQUESTED** [review]
make required changes.

**Comment by sidntrivedi** [comment]
Made the change.

**Review by khanchi97 — APPROVED** [review]
Thanks for your PR @sidntrivedi012 🎉

**Comment by sidntrivedi** [comment]
Thanks @khanchi97

**Pull request merged**